src/lib/publish.js

```diff
@@ -38,7 +42,23 @@ async function publishBatch({ queue, messages, logger }) {
   }
 }
 
+/**
+ * publish a message to a notification topic
+ * @param topic {string} - topic to publish message to, e.g. an SNS ARN
+ * @param message - message to send
+ * @param {SNSClient} client - SNS client to issue command to
+ * @returns Promise<void>
+ */
+async function notify({ client = snsClient, message, topic }) {
```

Comment:
is `notify` a different operation from `publish` from business logic perspective?

Reply:
why metrics are not implemented? if for some reason we don't want/need to do that, we should comment here

Reply:
there is no logs in case of failure, is it intentional?

Reply:
> is notify a different operation from publish from business logic perspective?

Kind of. `publish` assumes a job queue (sqs), and `notify` does a fire-and-forget pubsub publish/notify via SNS.

Reply:
> > is notify a different operation from publish from business logic perspective?
> 
> Kind of. `publish` assumes a job queue (sqs), and `notify` does a fire-and-forget pubsub publish/notify via SNS.

we should write this in comments or in docs, really hard to remember. This assumes that if the `notify` fails, the job is considered done. Is it fine to have a state where the car is published but not notified?